gen/gen.go

```diff
@@ -58,13 +57,14 @@ func init() {
 		dirMetadata = filepath.Join(path, "..", "..", "..", "x-common")
 	}
 	if _, path, _, ok := runtime.Caller(2); ok {
-		dirProblem = filepath.Join(path, "..")
+		dirExercise = filepath.Join(path, "..", "..")
 	}
-	if dirProblem == "" {
-		dirProblem = "."
+	if dirExercise == "" {
+		dirExercise = ".."
```

Comment:
unsure this really needs to change, but since we never get here for any current generator I can't complain since there is no correct answer nor incorrect answer